### PR TITLE
PLOW-99: Review job descriptions

### PR DESCRIPTION
**Pull request by bobek**
Clean up of existing job descriptions and clarification of certain roles. Especially product management, product owner and team leader.

Also redoing #131

### CODE REVIEW

**Comment by bobek** [comment]
Please, especially take a look at descriptions of Lead Engineer. I have renamed PO to PM and claimed, that LE/TL should do the job.

File: job-roles/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Roles at MayaData
+
+{{#include intro.md}}
+
+## Typicall Engineering Team
```
Comment (AmitKumarDas) [inline job-roles/README.md:5]: Spellcheck! Typical

Comment (bobek) [reply job-roles/README.md:5]: :facepalm:

Comment (bobek) [reply job-roles/README.md]: Thank you. Done some other spell checking as well.

File: job-roles/product-manager.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
... shape the future of Kubera and OpenEBS through building a vibrant contributor community. In this highly technical role, you will be responsible for specific MayaData subsystems and work with our Engineering and Product teams responsible for building and delivering them to market. You will be responsible for planning and delivering product content for new releases and launches of our offerings, prioritizing and communicating product requirements to guide development, supporting the field team in advanced sales opportunities, performing competitive reviews and analysis, and working with customers and partners. You will actively contribute to the overall company strategy and work closely with Engineering, Marketing, Support, Sales, and Field teams to ensure our offerings' success. You are also expected to engage with open source communities that support our container initiatives, including Kubernetes, other Cloud Native Computing Foundation (CNCF) projects, CSI, Velero, and more. As a Product Owner, you need to have great communication, teamwork, and persuasion skills. This is a great opportunity to work on a fast-growing offering alongside some of the brightest minds in open source.
```
Comment (GlennBullingham) [inline job-roles/product-manager.md:6]: This role is titled "Product Manager" but  the first sentence of the Role Description refers to "Product Owner".  Is this intentional?

File: job-roles/lead-engineer.md [hunk 3]
```diff
@@ -1,57 +1,51 @@
...
+- Understand the gaps by involving with multiple teams in company.
+- Suggest improvements and changes to the overall organization of the Engineering and your team.
+- Identify metrics for various components like code quality, team productivity, motivations, culture, sessions feedback etc.
+- Define goals/[OKRs](/process/okr.md) with and for your team.
+- Help you team to learn and grow, identifying external/internal training requirements.
+- Exhibit and inspire team to follow process/culture.
+- Track our marketing and sales activities. Encourage participation and participate at conferences, write blog articles, tweet etc.
+
+### Project Management
+- Keeping eye on your team's JIRA and keeping it healthy.
```
Comment (GlennBullingham) [inline job-roles/lead-engineer.md:23]: "Keeping an eye" is probably better grammar

File: job-roles/lead-engineer.md [hunk 4]
```diff
@@ -1,57 +1,51 @@
...
+- Project level check for required E2E coverage.
+- Make sure that we follow team's definition of done.
+
+### Product Owner
+- Convert Stories to backlog Epics and Tasks. Participate at grooming sessions with Product Management.
+- Be a close partner to Product Management organization to identify new initiatives, improve existing features and mitigate issues our customers may be facing.
+- Prioritize work for team according to agreed priorities with Product Management and the overall company goals and vision.
+- Understand market trends on technologies, products. Research competitive solutions.
+- Keep our persona needs on your mind. Proactively suggest changes to features or add new ones to backlog for discussion with Project Management.
+- Alwats make sure, that you know **why** we are building something, so you can explain to your colleagues.
```
Comment (GlennBullingham) [inline job-roles/lead-engineer.md:37]: typo: "Alwats" -> "Always"

**Review by GlennBullingham — APPROVED** [review]

File: job-roles/lead-engineer.md [hunk 5]
```diff
@@ -1,57 +1,51 @@
...
+- Keeping an eye on your team's JIRA and keeping it healthy.
+- Project level tracking for execution completion within time.
+- Lead/partcipate in internal project review meeting.
+- Helping in risk mitigation for project execution.
+- Plan team size in advance.
+- Project level check for required E2E coverage.
+- Make sure that we follow team's definition of done.
+
+### Product Owner
+- Convert Stories to backlog Epics and Tasks. Participate at grooming sessions with Product Management.
```
Comment (kmova) [inline job-roles/lead-engineer.md:32]: Should we call out here if this includes the ownership of meeting release milestones?

Comment (bobek) [reply job-roles/lead-engineer.md:32]: OK, let me propose something here.

Comment (kmova) [reply job-roles/lead-engineer.md:34]: cool!

**Review by kmova — APPROVED** [review]

**Review by muratkars — APPROVED** [review]

**Pull request merged**